### ISSUE TITLE
feat: apply paste_command and execute_command to visual selection

### ISSUE THREAD

**Issue by mikesmithgh** [mikesmithgh/kitty-scrollback.nvim#77]
Apply `paste_command` and `execute_command` to visual selection.
```
  vim.keymap.set({ 'n', 'i' }, '<Plug>(KsbExecuteCmd)', ksb_api.execute_command, { buffer = bufid })
  vim.keymap.set({ 'n', 'i' }, '<Plug>(KsbPasteCmd)', ksb_api.paste_command, { buffer = bufid })
```

e.g., user visually selects content in scrollback buffer and presses `<S-CR>` should paste the visual selection to Kitty.

This removes an extra step of yanking to the paste window and then pressing `<S-CR>`

**Comment by tummetott** [comment]
Yes I had exactly the same idea while configuring this plugin

**Comment by mikesmithgh** [comment]
Hey @tummetott, I just added this! Let me know if you have any issues.

**Comment by mikesmithgh** [comment]
closed by #134
